vars/karmaExecuteTests.groovy

```diff
@@ -89,6 +89,9 @@ void call(Map parameters = [:]) {
 
         utils.pushToSWA([step: STEP_NAME], config)
 
+        config.dockerEnvVars = config.dockerEnvVars ?: [:]
+        config.dockerEnvVars.PIPER_STEP_KARMA_EXECUTE_TESTS = "true"
```

Comment:
What are `PIPER_STEP_SELENIUM_EXECUTE_TESTS` and `config.dockerEnvVars.PIPER_STEP_KARMA_EXECUTE_TESTS` for? 
Shouldn't there be one for UIVeri5 as well?

Reply:
The idea is to have them as a flag so that scripts can determine in what kind of context they are called. E.g. in my `karma.conf.js`, if the `PIPER_STEP_KARMA_EXECUTE_TESTS` env var is not set, there is no need to apply any Piper-specific browser settings.

What do you think? 

I agree that if we do this, we should also set it for UIveri5...

Reply:
✅ Done

Reply:
Maybe it's not that explicit but can't we already use `PIPER_DOCKER_IMAGE` for this?

Reply:
Ok, fair enough then. Let's go without these env vars for now, and if we see the need to add them, we can still do that later.

Reply:
✅ Done